python/pp_ser.py

```diff
@@ -27,100 +33,97 @@
 """
 
 # information
-__author__    = 'Oliver Fuhrer'
+__author__ = 'Oliver Fuhrer'
 __copyright__ = 'Copyright 2014, MeteoSwiss'
-__license__   = 'GPL'
-__version__   = '0.1'
-__date__      = 'Sun Mar 23 22:06:44 2014'
-__email__     = 'oliver.fuhrer@meteoswiss.ch'
+__license__ = 'GPL'
+__version__ = '0.1'
+__date__ = 'Sun Mar 23 22:06:44 2014'
+__email__ = 'oliver.fuhrer@meteoswiss.ch'
 
-# modules
-import sys, os, tempfile, re, filecmp, shutil
 
-def toASCII(text):
+def to_ascii(text):
```

Comment:
This is a function I added to support both versions of Python. Do you know if there is a better way to achieve this? Also, do you know if ppser works correctly with both?